library/dd-sdk-android-rum/src/main/kotlin/com/datadog/android/rum/internal/domain/event/RumEventMapper.kt

```diff
@@ -32,6 +34,8 @@ internal data class RumEventMapper(
     val telemetryConfigurationMapper: EventMapper<TelemetryConfigurationEvent> = NoOpEventMapper()
 ) : EventMapper<Any> {
 
+    internal var sdkCore: SdkCore = NoOpSdkCore()
```

Comment:
we can inject it in the constructor, when we create a `RumDataWriter` in `RumFeature`